packages/lesswrong/components/comments/CommentsItem/CommentsItem.jsx

```diff
@@ -180,11 +182,20 @@ class CommentsItem extends Component {
               <Components.CommentsVote comment={comment} currentUser={currentUser} />
               {this.renderMenu()}
             </div>
-            <Components.CommentBody
-              truncated={truncated}
-              collapsed={collapsed}
-              comment={comment}
-            />
+            { showEdit ? (
```

Comment:
Seems like you can just omit the parentheses here, any reason to keep them?

Reply:
I find them a lot easier to read.

Reply:
K